packages/web/src/common/components/Select/index.tsx

```diff
@@ -1,9 +1,12 @@
 import isPropValid from "@emotion/is-prop-valid";
 import React, { useState, useRef, useEffect } from "react";
 import styled, { css } from "styled-components";
-import { KeyboardArrowDown, KeyboardArrowUp } from "@mui/icons-material";
-import Label from "./_atomic/Label";
-import ErrorMessage from "./_atomic/ErrorMessage";
+import Label from "../FormLabel";
+import FormError from "../FormError";
+import Icon from "../Icon";
+import NoOption from "./_atomic/NoOption";
+import Dropdown from "./DropDown";
```

Comment:
case 통일이 안 되어 있어요! Dropdown으로 통일하면 어떨까욤

Reply:
Already included file name '/Users/woojeong/SPARCS/ar-002-clubs/packages/web/src/common/components/Select/Dropdown.tsx' differs from file name '/Users/woojeong/SPARCS/ar-002-clubs/packages/web/src/common/components/Select/DropDown.tsx' only in casing.
  The file is in the program because:
    Imported via "./Dropdown" from file '/Users/woojeong/SPARCS/ar-002-clubs/packages/web/src/common/components/Select/index.tsx'
    Imported via "@sparcs-clubs/web/common/components/Select/Dropdown" from file '/Users/woojeong/SPARCS/ar-002-clubs/packages/web/src/common/components/Filter/_atomic/FilterDropdown.tsx'
    Matched by include pattern '**/*.tsx' in '/Users/woojeong/SPARCS/ar-002-clubs/packages/web/tsconfig.json'ts(1261)
    
이러한 에러가 뜨는군

Reply:
므엥 저게 모꼬

Reply:
아 단순 파일명 case 변경은 git diff로 안 잡히네??????

Reply:
https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git
요것두 한번 해볼래??? 나머지 변경사항은 add하고 얘만 mv로 인식시켜바

Reply:
이러면 변경사항 잡히는거같기두
<img width="1151" alt="image" src="https://github.com/academic-relations/ar-002-clubs/assets/33417582/0a7371fa-709a-4db9-9e93-c70ec453b3a1">